src/constants/redux.js

```diff
@@ -16,6 +16,7 @@ export type StateType = {
   +wallet: {
     +daiBalance: ?number,
     +xdaiBalance: ?number,
+    +walletAddress: ?string,
```

Comment:
`ethAddress`

Reply:
also, why do we need it in Redux state? 🤔 We could just use the one from Firestore

Reply:
💥